campaigns.json

```diff
@@ -2705,16 +2705,25 @@
     "go": true,
     "lumisize": -1,
     "maxcopies": 1,
-    "SecondaryLocation": [
-       "T1_IT_CNAF_Disk",
-       "T1_US_FNAL_Disk"
-    ],
     "SiteBlacklist": [
        "T2_CH_CERN",
        "T2_CH_CERN_HLT"
     ],
     "secondaries": {
-      "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL18_106X_upgrade2018_realistic_v16-v1/PREMIX": {}
+      "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL18_106X_upgrade2018_realistic_v16-v1/PREMIX": 
+      {
+              "SecondaryLocation": [
+                 "T1_IT_CNAF_Disk",
+                 "T1_US_FNAL_Disk"
+               ]
+      },
+      "/Neutrino_E-10_gun/RunIIFall17FSPrePremix-PUFSUL18CP5_106X_upgrade2018_realistic_v16-v1/PREMIX":
```

Comment:
This one is only at FNAL. I don't think RAL is a good choice for EU site. It has too many issues with transfers. How about French T1?

Reply:
You mean this one: T1_FR_CCIN2P3_Disk

Reply:
How about updating the TR ticket? https://its.cern.ch/jira/browse/CMSTRANSF-300 Replacing RAL w/ CCIN2P3?

Reply:
Sure I wanted to check this was this site you meant before updating it.

Reply:
Thanks Scarlet.